intellst-api/config/routes.yaml

```diff
@@ -1,3 +1,12 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
+register:
```

Comment:
it is better to configure import of routing by annotations in controllers instead of defining each route here